frontend/src/components/accoitem.jsx

```diff
@@ -1,22 +1,34 @@
 import { Col, Image, OverlayTrigger, Popover } from "react-bootstrap";
 import { Link } from "react-router";
 
-export default function AccoItem({ iden, name, body, foot, shot }) {
+import { rareColors } from "../features/util.js";
+
+export default function AccoItem({ iden, name, body, foot, shot, rare }) {
   return (
     <OverlayTrigger
       placement="auto"
       overlay={
         <Popover className="bodyelem">
-          <Popover.Header className="p-2">{name}</Popover.Header>
+          <Popover.Header
+            className="p-2 fw-bold text-truncate"
+            style={{ background: rareColors[rare] || "#6c757d", color: "#ffffff" }}
```

Comment:
It would be more efficient to define CSS classes for the rarities and apply the classes, rather than do it in Javascript. As a rule of thumb CSS is a better place for styling, while JS is a good place for interactions.
Also, there are [Content Security Policy restrictions](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Security-Policy/style-src-attr) that limit the use of `style` attributes in HTML tags, I'm not sure which ones we use in fedoraproject.org at the moment but it's safer to limit the use of inline attributes when possible.